app/repos/exemption.js

```diff
@@ -122,9 +122,9 @@ const handleOrder2023 = (registration, data) => {
  */
 const setDefaults = (registration, data) => {
   if (
-    (registration.cdo_expiry === null || registration.cdo_expiry === undefined) &&
-    (registration.cdo_issued === null || registration.cdo_issued === undefined) &&
-    (data.cdoIssued !== null && data.cdoIssued !== undefined)
+    (data.cdoExpiry === null || data.cdoExpiry === undefined) &&
```

Comment:
Is it not easy to just write 
if (!data.cdoExpiry && !data.cdoIssued & etc
??

Reply:
was concerned here about it catching falsy values, prefer to be more explicit.  Not that I can foresee a falsy value being caught here.